core/src/main/scala/org/apache/spark/ml/param/PythonWrappableParam.scala

```diff
@@ -41,7 +41,7 @@ trait PythonWrappableParam[T] extends Param[T] {
 
   val name: String
 
-  type InnerType = T
+  type pyInnerType = T
```

Comment:
Will we need separate values per language?

Reply:
Actually no, and dotnet didn't used it.. I'll change it back!